drivers/docker/config.go

```diff
@@ -636,6 +645,14 @@ func (d *Driver) SetConfig(c *base.Config) error {
 		d.config.GC.DanglingContainers.CreationGrace = dur
 	}
 
+	if len(d.config.PullActivityTimeout) > 0 {
+		dur, err := time.ParseDuration(d.config.PullActivityTimeout)
+		if err != nil {
+			return fmt.Errorf("failed to parse 'pull_activity_timeout' duaration: %v", err)
+		}
```

Comment:
Let's add a minimum activity timeout too, at least to rule out negative and zero values.  Would 1 minute be an appropriate floor?

Reply:
That sounds good, thanks. I'll add that. New commits incoming soon.